lib/much-rails/action/base_router.rb

```diff
@@ -302,23 +304,30 @@ def empty?
     def add(name, path)
       url = @router.url_class.new(@router, path, name.to_sym)
       key = url.name
-      if !@set[key].nil?
-        raise ArgumentError, "There is already a URL named `#{name.to_sym.inspect}`."
+      unless @set[key].nil?
+        raise ArgumentError,
+              "There is already a URL named `#{name.to_sym.inspect}`."
```

Comment:
Prefer steps not flag-poles please:
```suggestion
        raise(
          ArgumentError,
          "There is already a URL named `#{name.to_sym.inspect}`."
        )
```